systest/audit_encrypted/audit_test.go

```diff
@@ -0,0 +1,245 @@
+		fmt.Printf("Output %v\n", string(out))
+		t.Fatal(err)
+	}
+
+	verifyLogs(t, fmt.Sprintf("audit_dir/za/zero_audit_0_%s.log", nId), msgs)
+}
+func TestAlphaAuditEncrypted(t *testing.T) {
+	state, err := testutil.GetState()
+	require.NoError(t, err)
+	var nId string
+	for key := range state.Groups["1"].Members {
+		nId = key
+	}
+	defer os.Remove(fmt.Sprintf("audit_dir/aa/alpha_audit_1_%s.log", nId))
+	defer os.Remove(fmt.Sprintf("audit_dir/aa/alpha_audit_1_%s.log.enc", nId))
+	testCommand := map[string][]string{
+		"/admin": {"--location", "--request", "POST",
+			fmt.Sprintf("%s/admin", testutil.SockAddrHttp),
+			"--header", "Content-Type: application/json",
+			"--data-raw", `'{"query":"mutation {\n  backup(
+input: {destination: \"/Users/sankalanparajuli/work/backup\"}) {\n    response {\n      message\n      code\n    }\n  }\n}\n","variables":{}}'`}, //nolint:lll
+
+		"/graphql": {"--location", "--request", "POST", fmt.Sprintf("%s/graphql", testutil.SockAddrHttp),
+			"--header", "Content-Type: application/json",
+			"--data-raw", `'{"query":"query {\n  __schema {\n    __typename\n  }\n}","variables":{}}'`},
+
+		"/alter": {"-X", "POST", fmt.Sprintf("%s/alter", testutil.SockAddrHttp), "-d",
+			`name: string @index(term) .
+			type Person {
+			  name
+			}`},
+		"/query": {"-H", "'Content-Type: application/dql'", "-X", "POST", fmt.Sprintf("%s/query", testutil.SockAddrHttp),
+			"-d", `$'
+			{
+			 balances(func: anyofterms(name, "Alice Bob")) {
+			   uid
+			   name
+			   balance
+			 }
+			}'`},
+		"/mutate": {"-H", "'Content-Type: application/rdf'", "-X",
+			"POST", fmt.Sprintf("%s/mutate?startTs=4", testutil.SockAddrHttp), "-d", `$'
+			{
+			 set {
+			   <0x1> <balance> "110" .
+			   <0x1> <dgraph.type> "Balance" .
+			   <0x2> <balance> "60" .
+			   <0x2> <dgraph.type> "Balance" .
+			 }
+			}
+			'`},
+	}
+
+	msgs := make([]string, 0)
+	// logger is buffered. make calls in bunch so that dont want to wait for flush
+	for i := 0; i < 200; i++ {
+		for req, c := range testCommand {
+			msgs = append(msgs, req)
+			cmd := exec.Command("curl", c...)
+			if out, err := cmd.CombinedOutput(); err != nil {
+				fmt.Println(string(out))
+				t.Fatal(err)
+			}
+		}
+	}
+	alphaMsgs = msgs
+
+	var args []string
+	args = append(args, "audit", "decrypt",
+		"--encryption_key_file=../../ee/enc/test-fixtures/enc-key",
+		"--in", fmt.Sprintf("audit_dir/aa/alpha_audit_1_%s.log.enc", nId),
+		"--out", fmt.Sprintf("audit_dir/aa/alpha_audit_1_%s.log", nId))
+
+	decrypt := exec.Command(testutil.DgraphBinaryPath(), args...)
+
+	fmt.Println("Running: ", decrypt.Args)
+
+	out, err := decrypt.CombinedOutput()
+	if err != nil {
+		fmt.Printf("Error %v\n", err)
+		fmt.Printf("Output %v\n", string(out))
+		t.Fatal(err)
+	}
+
+	verifyLogs(t, fmt.Sprintf("audit_dir/aa/alpha_audit_1_%s.log", nId), msgs)
+}
+
+// deprecated logs generated with dgraph v22.0.2
+func TestZeroAuditDecryptDeprecated(t *testing.T) {
+	if len(zeroMsgs) == 0 {
```

Comment:
I think this is not necessary, the test could run in either order. Each test should be independent.

Reply:
done